src/packages/core/workspace/workspace-property-layout/workspace-property-layout.element.ts

```diff
@@ -80,18 +86,28 @@ export class UmbWorkspacePropertyLayoutElement extends LitElement {
 				grid-column: span 2;
 			}
 
+			:host > div:last-child {
```

Comment:
This selector feels unsafe to me.
When reading it, one does not easily capture what its suppose to do. Therefor is some one day comes a head and adds another div then this would break.

My solution on this one is to add an ID to the div, like #editorColumn.